dart/lib/src/sentry.dart

```diff
@@ -360,7 +360,11 @@ class Sentry {
   /// Gets the current active transaction or span bound to the scope.
   static ISentrySpan? getSpan() => _hub.getSpan();
 
-  static Future<void> addFeatureFlag(String name, bool value) async {
+  static Future<void> addFeatureFlag(String name, dynamic value) async {
```

Comment:
If we allow dynamic users could set whole objects, or would we check for `string`, `int` type etc?

Reply:
we will check for the type in the future, this is how JS does it as well https://github.com/getsentry/sentry-javascript/blob/56aac537ea2627baec9025291b73f46b48670abf/packages/browser/src/utils/featureFlags.ts#L59C1-L66C1